%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Document: Thesis for PhD at UC Riverside                          %%
%% Description: A comparative analysis of environment sensing in EDF %%
%% Author: Steven Ahrendt                                            %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% RHODOPSIN FIGURES %
%%%%%%%%%%%%%%%%%%%%%
% Photosensory protein distribution in Fungi
\begin{figure}[hb]
  \centering
  \includegraphics[width=4in]{./Chapter_RhodStruct/img/SpStructure.png}
  \caption[Structural details of the \textit{S. punctatus} chytriopsin homology model.]{Key \textit{S. punctatus} residues are colored according to function: orange (“binding pocket” residues), red (putative counterion), purple (disulfide bond), yellow (salt bridge), dark blue (NPxxY motif), and pink \& black (ion lock). Light purple functional and backbone residues belong to \textit{T. pacificus}, while grey backbone residues belong to \textit{S. punctatus}. The ideal position of the 11-\textit{cis}-retinal ligand, taken from the \textit{T. pacificus} crystal structure, is shown in green. A) C$\alpha$ backbone structural alignment of \textit{S. punctatus} homology model and \textit{T. pacificus} x-ray crystal structure, displayed in a cartoon representation, with key residues displayed as sticks. B) Topography plot of membrane spanning regions of \textit{S. punctatus} homology model. C) Detail of \textit{S. punctatus} binding pocket residues aligned with those of \textit{T. pacificus}. The lysine residue from \textit{S. punctatus} (cyan) is present in an optimal position to interact with the 11-\textit{cis}-retinal ligand (green), an aspect unique among the chytriopsins studied. The putative \textit{S. punctatus} counterion D94 (red) is also situated in an ideal position for photoisomerization. D) Detail of \textit{S. punctatus} disulfide bond (purple) and salt bridge (yellow) regions aligned with those of \textit{T. pacificus}. The view is from the top (extracellular side) of the protein, into the 11-\textit{cis}-retinal (green) binding pocket. E) Detail of \textit{S. punctatus} ERY and NPxxY regions aligned with those of \textit{T. pacificus}. In \textit{T. pacificus}, residues R133 and E256 (light purple) form a salt bridge in the inactive state to hold the structure in place, which is broken upon receptor activation. The corresponding \textit{S. punctatus} residues R116 (pink) and E250 (black) are in equivalent positions and presumably carry out the same function.}
  \label{fig:ChRhod_SpStructure}
\end{figure}

%\begin{figure}[hb]
%  \centering
%  \includegraphics[width=4in]{./Chapter_Rhodopsin/img/BdStructure.png}
%  \caption[Structural details of the \textit{\textit{B. dendrobatidis}} chytriopsin homology model.]{Key \textit{B. dendrobatidis} residues are colored according to function: orange (“binding pocket” residues), red (putative counterion), purple (disulfide bond), yellow (salt bridge), dark blue (NPxxY motif), and pink \& black (ion lock). Light purple functional and backbone residues belong to \textit{T. pacificus}, while grey backbone residues belong to \textit{B. dendrobatidis}. The ideal position of the 11-cis-retinal ligand, taken from the \textit{T. pacificus} crystal structure, is shown in green. A) C$\alpha$ backbone structural alignment of \textit{B. dendrobatidis} homology model and \textit{T. pacificus} x-ray crystal structure, displayed in a cartoon representation, with key residues displayed as sticks. B) Topography plot of membrane spanning regions of \textit{B. dendrobatidis} homology model. C) Detail of \textit{B. dendrobatidis} binding pocket residues aligned with those of \textit{T. pacificus}. D) Detail of \textit{B. dendrobatidis} disulfide bond (purple) and salt bridge (yellow) regions aligned with those of \textit{T. pacificus}. The view is from the top (extracellular side) of the protein, into the 11-cis-retinal (green) binding pocket. E) Detail of \textit{B. dendrobatidis} ERY and NPxxY regions aligned with those of \textit{T. pacificus}.}
%  \label{fig:ChRhod_BdStructure}
%\end{figure}

%\begin{figure}[hb]
%  \centering
%  \includegraphics[width=4in]{./Chapter_Rhodopsin/img/AmStructure.png}
%  \caption[Structural details of the \textit{\textit{A. macrogynus}} chytriopsin homology model.]{Key \textit{A. macrogynus} residues are colored according to function: orange (“binding pocket” residues), red (putative counterion), purple (disulfide bond), yellow (salt bridge), dark blue (NPxxY motif), and pink \& black (ion lock). Light purple functional and backbone residues belong to \textit{T. pacificus}, while grey backbone residues belong to \textit{A. macrogynus}. The ideal position of the 11-cis-retinal ligand, taken from the \textit{T. pacificus} crystal structure, is shown in green. A) C$\alpha$ backbone structural alignment of \textit{A. macrogynus} homology model and \textit{T. pacificus} x-ray crystal structure, displayed in a cartoon representation, with key residues displayed as sticks. B) Topography plot of membrane spanning regions of \textit{A. macrogynus} homology model. C) Detail of \textit{A. macrogynus} binding pocket residues aligned with those of \textit{T. pacificus}. D) Detail of \textit{A. macrogynus} disulfide bond (purple) and salt bridge (yellow) regions aligned with those of \textit{T. pacificus}. The view is from the top (extracellular side) of the protein, into the 11-cis-retinal (green) binding pocket. E) Detail of \textit{A. macrogynus} ERY and NPxxY regions aligned with those of \textit{T. pacificus}. In \textit{T. pacificus}, residues R133 and E256 (light purple) form a salt bridge in the inactive state to hold the structure in place, which is broken upon receptor activation.}
%  \label{fig:ChRhod_AmStructure}
%\end{figure}
