%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Document: Thesis for PhD at UC Riverside                                    %%
%% Title: Investigating the evolution of environmental and biotic interactions %%
%%          in basal fungal lineages through comparative genomics              %%
%% Author: Steven Ahrendt                                                      %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% INTRODUCTION %
%%%%%%%%%%%%%%%%
\chapter{Introduction to the Basal Fungal lineages}
\section{Overview of fungal phylogenetics and the importance of the basal lineages}
%%%%%
% Fungal phylogenetics
%%%%%
The Fungi are one of the major kingdoms of eukaryotic life on
Earth. Various studies have attempted to estimate the date of the
emergence of the Kingdom Fungi \cite{Taylor2006}, when it diverged
from the metazoan lineages. These studies place this occurance at
approximately 1 billion years ago, with a range of around $\pm$500
MYA: 600 MYA \cite{Berbee1993}, 965 MYA \cite{Doolittle1996}, and 1.6
BYA \cite{Wang1999}. These loose approximations are based on
correlation between evolutionary events in fungi and in other
organisms, and are under continued re-evaluation and refinement
\cite{Berbee2010}.\\
\indent A comprehensive review of a collection of 21st century
phylogenetic studies \cite{Hibbett2007} proposes that the Fungal
Kingdom comprises seven phyla: Microsporidia, Chytridiomycota,
Blastocladiomycota, Neocallimastigomycota, Glomeromycota,
Basidiomycota, and Ascomycota, with the most recent inclusion of the
phylum Cryptomycota \cite{Jones2011}.\\
\indent The most recently diverged fungal phyla are the Ascomycota and
Basidiomycota, and together make up the subkingdom Dikarya. This
subkingdom is so named because its members undergo cell fusion
(plasmogamy) without nuclear fusion (karyogamy) during sexual
development, resulting in cells with nuclei from individual parents
("dikaryons"). These organisms are muticellular, with terrestrial
habitats, sexual and asexual life cycle components, and filamentous
growth structures. Collectively, the Dikarya is the most widely
studied group and is home to several model and non-model organisms of
research interest, including the model filamentous Ascomycete
\textit{Neurospora crassa}, the economically critical
\textit{Saccharomyces cerevisiae}, and the numerous medically relevant
\textit{Aspergillus} spp.\\
\indent Going further back in time is the phylum Glomeromycota. This
group contains mycorrhizal fungi (arbuscular and ento-) which
associate with approximately 90\% of all plant species and are thus of
great ecological importance. This phylum also contains four subphyla
\textit{incertae sedis}: Mucormycotina, Entomophthoromycotina,
Zoopagomcotina, and Kickxellomycotina \cite{White2006}. These subphyla
were previously classified in the phylum Zygomycota and contain
primarily terrestrial fungi of various medical and industrial research
interest. It is important to point out here that the definitive
phylogenetic relationship between the Glomeromycota and Zygomycota
\textit{incertae sedis} lineages is unresolved and is a current focus
of research \cite{Hibbett2007}. Therefore in this text, the
nonflagellated members of the Glomeromycota, Mucormycotina,
Entomophthoromycotina, Zoopagomycotina, and Kickxellomycotina will,
for convenience, be referred to collectively as Zygomycota.\\
\indent Closest to the fungal-animal evolutionary divergence are the
basal fungal lineages: the Microsporidia, Cryptomycota,
Chytridiomycota, Blastocladiomycota, and Neocallimastigomycota. It is
these lineages, particularly the Blastocladiomycota and
Chytridiomycota, on which this dissertation will primarily be
regarding. These groups are sometimes collectively referred to as
"chytrids", although this is not to be confused with the formal phylum
Chytridiomycota. Broadly speaking, these organisms have asexual life
cycles which progress through development as motile, flagellated
zoospores, followed by sessile, non-flagellated, spore-producing
sporangia. During the motile stage, the zoospore seeks an appropriate
environmental substrate, encysts upon it, retracts its flagellum, and
develops a cell wall. Many species will stay dormant in this stage as
a thick-walled "resting spore", and only develop into a thin-walled
zoosporangia after a certain time period \cite{James2006Blasto}. Other
chytrid species will instead progress directly to the zoosporangia
stage, undergo several rounds of mitotic cell division, and ultimately
produces and releases hundreds of new zoospores
\cite{James2006Blasto}. \\
\indent While generally being described as asexual, certain species
within the Blastocladiomycota, such as \textit{Allomyces reticulatus}
and \textit{Coelomomyces punctatus}, have demonstrated sexual
reproductive cycles utilizing zoospores of different mating types
\cite{Alexopoulos1996}.\\

%%%%%
% Importance of EDF
%%%%%
\indent The basal fungal lineages are characterized as true Fungi 
and distinct from other water molds, like Oomycetes, and fall sister 
to both Metazoan lineages as well as the other fungal lineages 
(Zygomycota, Ascomycota, and Basidiomycota). While these lineages 
only represent less than 2\% of all described fungi \cite{Stajich2009}, 
they serve as a unique system in which to infer characteristics presumed 
to have been present in the fungal-animal common ancestor.\\
\indent Additionally, basal fungal lineages are presumed to have a nearly 
cosmopolitan distribution \cite{Powell1993}. Members of these lineages 
are found in widespread environments \cite{Tedersoo2014}, and in some
biomes represent the dominant member of the soil fungal community 
\cite{Freeman2009}. Chytrids fulfill nearly all varieties of ecological niches, primarily 
decomposition in terrestrial and aquatic environments, but also including 
pathogenic interactions with a wide variety of hosts: arbuscular mycorrhizal 
fungi (\textit{Spizellomyces punctatum} (Chytridiomycota) \cite{Paulitz1984}), 
insects (\textit{Coelomomyces psorophorae} (Blastocladiomycota) \cite{Zebold1979}), 
plants (\textit{Olpidium brassicae} (Chytridiomycota) \cite{Tewari1983}), 
vertebrates (\textit{Batrachochytrium dendrobatidis} (Chytridiomycota) \cite{Longcore1999}), 
nematodes (\textit{Catenaria anguillulae} (Blastocladiomycota) \cite{Deacon1997}), 
algae (\textit{Zygorhizidium plantonicum} \cite{Canter1967}), and even intracellular 
symbioses with other chytrids (\textit{Rozella allomyces} (Cryptomycota) \cite{Held1973}). 
This distribution of life styles speaks to the vast biological 
challenges they must face and therefore suggests a number of novel 
mechanisms which have yet to be fully studied and explored. \\
\indent Attempts at formal description of early-diverging fungi, 
based primarily on collection and observation, began as early as 
1858 and proceeded through the latter half of 19th century with 
pioneering work of researchers such as Schroter, Fischer, Zopf, 
Lowenthal, Nowakowski, and Woronin \cite{Fitzpatrick1930}. 
A primarily systematic approach allowed for the establishment of 
(among others) the order Chytridiales, defined broadly as lacking 
mycelium and having an unknown sexual cycle, and the order Blastocladiales, 
defined as having mycelium and a sexual reproductive component.\\
\indent Significant microscopy work was carried out on chytrid species
as early as 1953 by William Koch. This allowed for the discussion
of zoosporic ultrastructure characters \cite{Koch1958motileI,Koch1958motileII}
and the description of 6 major cell morphologies \cite{Koch1961motileIII},
which demonstrated a high degree of structural diversity among the zoosporic lineages. \\
\indent With the advent of nucleic acid-based molecular phylogenetic techniques
at the turn of the 21st century, several researchers addressed revisions of
fungal phylogenetics and attempted to reconcile traditional ultrastructure-based
phylogenetic ideas with these modern techniques. Whereas earlier, characters such
as zoospore discharge, thallus development, and ultrastructural features were used
to place basal lineages, modern ssu rDNA techniques corroborated these placements.
Modern techniques also supported the monophyletic nature of the Fungi and metazoan
lineages, as well as the inter-kingdom relationships between the fungal phyla. This
and other work helped firmly establish the Chytridiomycota as one of the four major
fungal phyla alongside the Zygomycota, Basidiomycota, and Ascomycota
\cite{Bruns1991,Bruns1992,Wainright1993,James2006sixGene,James2006Blasto,Hibbett2007}.\\
\indent It was around that time that the Chytridiomycete
\textit{Batrachochytrium dendrobatidis} emerged as a global pathogen
and the accepted causative agent of worldwide amphibian decline
\cite{Berger1998}. This emergence renewed interest in chytrids 
as a general system of study as emphasis was placed on understanding the distribution, 
diversity, and pathogenicity of \textit{Bd}.\\
\indent Despite widespread distribution in both geography and ecology,
and the existence of species pathogenic in a wide range of hosts,
chytrids remain understudied as a whole, driven in part because few
species are of substantial economic importance
\cite{Powell1993,James2000} as well as moderate difficulty in
collection and culturing methods. However, a greater molecular 
understanding of the phylogeny is a current subject of study. 
Within the past decade, formal descriptions for two new genera, 
the Irineochytrium (Chytridiomycota; Chytridiales) \cite{Letcher2014} 
and Fayochytriomyces (Chytridiomycota; Chytridiales) \cite{Davis2015}, 
and one new order, the Loulomycetales (Chytridiomycota) \cite{Simmons2009} 
have been published.\\

%%%%%
% Chytrid Bioinformatic resources 
%%%%%
\section{History of bioinformatic resources for basal fungi}
As whole genome sequencing continues to become more widely accesible,
chytrid genomes are more easily developed. The first available
bioinformatic resource was an expressed sequence tag (EST) dataset
published in 2005 for the Blastocladiomycete \textit{Blastocladiella
  emersonii} \cite{Ribichich2005}. This collection of 16,984
high-quality ESTs provided a first approach to understanding gene
complexity in chytrids. In 2006, a draft assembly for the genome of
\textit{B. dendrobatidis} strain JEL423 was made publically available
through the Broad Institute Fungal Genome Initiative
(\url{http://www.broadinstitute.org/}). The resulting assembly is 23.72 Mb,
represents 7.4X coverage of this diploid strain, and was the first
whole genome assembly for any chytrid. In 2008, a second draft genome
was released for \textit{B. dendrobatidis} strain JAM81 through the
Joint Genome Institute
(\url{http://genomeportal.jgi-psf.org/Batde5/Batde5.home.html}). This
assembly is 24.3 Mb and represents 8.74X coverage. \\
\indent As part of the push to understand the Origins of
Multicellularity \cite{RuizTrillo2007}, the genomes for 
\textit{Allomyces macrogynus} (Blastocladiomycete) and 
the exclusively terrestrial \textit{Spizellomyces punctatus} 
(Chytridiomycete) were sequenced by the Broad Institute in 2009. \\
\indent In 2011, the non-pathogenic species \textit{Homolaphlyctis
  polyrhiza} strain JEL142, the closest identified relative to
\textit{Bd} was sequenced for comparision to try to identify aspects
of pathogenicity in \textit{Bd} \cite{Joneson2011}. This aquatic chytrid 
has been isolated only once and while it had been used in previous phylogenetic 
\cite{James2000,James2006sixGene,Letcher2008} studies was only recently provided with a formal name \cite{Longcore2011}. 
The resulting assembly generated from 454 sequencing technology was 26.7 Mb
(haploid) from 11.2X coverage. In 2014, with the help of a postdoctoral researcher in the
Stajich Lab, Dr. Peng Liu, I generated Illumina sequencing libraries
for \textit{H. polyrhiza} and assisted in assessing the assembly and
annotation with Dr. Stajich; the results of which are described in
Chapter~\ref{chap:HpInhibition}.\\
\indent \textit{Gonapodya prolifera} (Monoblepharidomycota) is an
aquatic fungus with both sexual and asexual reproductive schemes, and
encompassing both hyphal and zoosporic growth stages. In the
environment, \textit{G. prolifera} is an active degrader of plant
material \cite{Karling1977}. A draft genome assembly was made available in
2011 through JGI, with the goal of identifying potentially novel
degredation-related enzymes for biofuels applications. \\
\indent \textit{Catenaria anguillulae} (Blastocladiomycota) is a
facultative parasite of nematodes \cite{Deacon1997}. A draft genome
was made available in 2010 by the JGI and was the second
Blastocladiomycete genome (after \textit{A. macrogynus}). Genomic
resources for \textit{C. anguillulae} allow for research into
monitoring and potential remediation strategies as the nematodes upon
which it parasitizes are themselves parasites of agriculturally
important crops.\\
\indent Members of the Neocallimastigomycota lineage, first isolated
and described in 1975, are found in the anaerobic environment of
mammalian rumen \cite{Orpin1975}. These fungi are uniquely adapted to
degradation of the high fiber content of the typical diets of cattle
and sheep. Thus they are important models for potential manipulation
to not only improve digestion in these livestock sources \cite{Ho1995}
but also potential biofuels applications
\cite{Youssef2013,Gruninger2014}. \textit{Piromyces} and \textit{Orpinomyces} are
two members of this group and were sequenced in 2011 and 2013,
respectively, in the hopes that understanding the genomic content
would provide starting points for these applications. \\
\indent The genome of Cryptomycete \textit{Rozella allomycis}, the
intracellular parasite of \textit{Allomyces}, was sequenced in 2013
\cite{James2013} and the analysis used to propose a unification of the
Cryptomycota and Microsporidian lineages. I assisted in this work 
by analysing a comparative search of flagellar-associated proteins
and conservation across the fungal lineages. A summary of this contribution
is provided in Appendix~\ref{app:Flagella}.\\
\indent In 2014 the first transcriptome of the mosquito pathogen
\textit{Coelomomyces lativitattus} (\textit{Cl}) was generated by
isolating RNA from gametes emerging from copepods. The RNA extraction
and Illumina library preparation was performed by Rob Hice, a
researcher in Dr. Brian Federici's lab at UCR. The sequencing was
performed at the UCR IIGB Genomics core. The resulting sequence data
was assembled and annotated using scripts provided by Dr. Jason
Stajich, and my analysis is described in
Chapter~\ref{chap:ClatTranscriptome}.\\
\indent The near future of bioinformatics resources for the basal
lineages is promising due to the efforts of the 1000 Fungal Genome
Project (\url{http://1000.fungalgenomes.org}) with plans to sequence the 
genomes of additional fungi from the Chytridiomycota and Blastocladiomycota 
lineages. A total of 16 Chytridiomycota species have been nominated for 
sequencing, including \textit{Operculomyces laminatus} JEL223, 
\textit{Rhizoclosmatium hyalinus} JEL800, and \textit{Obelidium mucronatum} JEL802, 
which are being prepared by the Stajich lab and were used in the experiments 
described in Chapter~\ref{chap:HpInhibition}. \textit{Coelomomyces lativittatus}, 
the mosquito pathogen for which a transcriptome analysis is presented in 
Chapter~\ref{chap:ClatTranscriptome} is being prepared for sequencing by 
Dr. Brian Federicis' lab at UCR and will be the third Blstocladiomycte genome
produced after \textit{A. macrogynus} and \textit{C. anguillulae}.\\

%%%%%
% Research goals
%%%%%
\section{Hypotheses and Objectives}
\indent Rapid advances in the feasability of genome sequencing have
yielded and will continue to yield an increasing number of fungal
genomes, especially those in the early-diverging lineages, for
comparative analyses. While incorporating comparisons to already
well-characterized representative fungal groups, this dissertation
work gives specific focus to members of the early-diverging lineages,
and in particular to the ones for which genomic resources are
available and which have obvious economic or ecological importance:
the amphibian pathogen \textit{Batrachochytrium dendrobatidis} and its
closest, non-pathogenic relative, \textit{Homolaphlyctis polyrhiza};
the saprobic \textit{Spizellomyces punctatus}; and the aquatic
Blastocladiomycete \textit{Allomyces macrogynus} and related
\textit{Catenaria anguillulae}. This thesis is presented in four
chapters comprising three aspects of biology focused on sensing and
interpretation of biotic and environmental signals.\\
\subsection*{Competition-based secondary metabolism and anti-fungal properties of \textit{Homolaphlyctis polyrhiza}}
Interactions between microorganisms are facilitated by biological
signals. These include proteins, small molecules, and various chemical
compounds, either bound to the cell surface or secreted into the
environment. Many of these compounds can be classified as secondary
metabolites: chemicals not required for growth or development of the
organism.\\
\indent Resource competition likely plays a role in the
evolution of natural antifungal production
\cite{Vicente2003}. Secretion by an organism in a resource-limited
environment of secondary metabolites which also happen to negatively
impact neighboring organisms would confer a selective advantage upon
the producer. \\
\indent Comparative genomic analyses have identified a host of degradation enzymes in basal fungi, suggesting saprotrophic and sometimes pathogenic associations with other organisms. There are few explored examples of secreted or secondary metabolite molecules produced by any of the zoosporic fungi.\\ 
\indent Secondary metabolite production as it applies to basal fungi is discussed in more detail in Chapter~\ref{chap:HpInhibition} using the non-pathogenic Chytridiomycete \textit{Homolaphlyctis polyrhiza} JEL142. In this chapter, I address three major questions regarding an initial observation I made in the Stajich lab of \textit{Hp} inhibition of the vegetative hyphal growth of \textit{N. crassa} via an unknown secreted compound. Namely, "Is Hp unique among the chytrids in this behavior?", "Is this behavior specific to \textit{N. crassa}?", and "What is the underlying biochemical mechanism by which this behavior is accomplished?". These questions are addressed using observational assays with the sporangia of related Chytridiomycetes, and probing the breadth of non-Chytridiomycete fungi whose growth is susceptible to \textit{Hp}, encompassing Ascomycete, Basidiomycete, and Zygomycete species, and including both temperature and proteinase screens. Finally, to better explore \textit{Hp} gene content, I produced an improved genome assembly and annotation, by assisting Dr. Peng Liu and Dr. Jason Stajich in the collection of fungal material and the assembly and annotation of the resulting genome sequence.\\
\subsection*{Mechanics and evolution of Fungal rhodopsin-based photosensing in the basal lineages}
\indent During the course of a given day, an organism experiences a
multitude of environmental stimuli, with light being one of the most
prominent. The biochemical ability to appropriately process and
respond to these signals is an incredibly complex and involved task,
and understanding the underlying mechanisms of these responses is an
ongoing scientific challenge. \\
\indent Previous work has shown that some of the basal fungi are
phototaxic (see \cite{Saranak1997} and \cite{Muehlstein1987}), however
the full extent of photosensing in zoosporic fungi has not been fully
explored. A recent review of fungal photobiology suggests a sporadic
distribution of photosensory proteins among the non-flagellated fungal
lineages (ie Zygomycota and Dikarya), with little emphasis placed on
the basal lineages \cite{Idnurm2010}. There are many classes of
photoreceptor proteins in fungi capable of producing a cellular
response from an environmental light signal, all of which have
different mechanisms of action and specializations: phytochromes,
cryptochromes, the white-collar complex, and opsins
\cite{Idnurm2010}. In plants, phytochromes function as day-night
sensors to regulate the circadian rhythm and flowering response. This
is accomplished through a conformational shift between the red and
far-red sensitive forms of the protein structure
\cite{Rockwell2006}. While relatively little is known about fungal
phytochrome function, research on \textit{A. nidulans} suggests that
the phytochrome protein is a member of an elaborate complex with
regulatory functions involved with the asexual-sexual transition and
secondary metabolite biosynthesis \cite{Idnurm2010}. Cryptochromes,
found predominantly in plants, animals, and insects, are blue-light
sensitive proteins involved in circadian rhythm regulation and light
activated DNA damage repair \cite{Idnurm2010}. Additional evidence
suggests cryptochrome proteins play a role in mediating the
phototactic behavior of sponge larvae \cite{Rivera2012}.\\
\indent First studied in the model filamentous Ascomycete fungus \textit{Neurospora crassa}, the well-characterized white-collar complex assembles as a heterodimer comprising White-collar 1 and 2 proteins. This complex functions to sense blue and near UV wavelengths, and, when active, directly interacts with DNA to regulate the circadian clock machinery, sporulation, pigmentation, and phototropism \cite{Ballario1997,Purschwitz2006,Corrochano2007}. \\
\indent The largest family of membrane receptors by far is that of the seven-transmembrane (7TM) receptors, comprising upwards of 800 genes \cite{Pierce2002}. This family includes various receptors for a wide range of ligands, including hormones, neurotransmitters, odorants, and photons. While there are three distinct subfamilies (A, B, and C), they share very little sequence similarity. Opsins, examples of which can be found in bacteria, archaea, and eukaryotes, are part of the largest family of 7TM proteins. One subclass of opsin, the Type 2 rhodopsins, are G protein-coupled receptor (GPCR) proteins which function via photoisomerization of a covalently bound retinyledene chromophore, typically 11-\textit{cis}-retinal \cite{Wald1968}.\\
\indent The retinal chromophore utilized in Type 2 rhodopsin-mediated photoreception is biosynthesized from $\beta$-carotene \cite{VonLintig2000}. Photoisomerization of this chromophore results in a conformational shift to the all-\textit{trans} isomer \cite{Smith2010} and activation of the coupled heterotrimeric G protein. A comparative analysis of auxillary proteins in basal fungi involved in this downstream signalling cascade is given in Chapter~\ref{chap:RhodAux}, and a description of findings dealing with structural and functional analyses of putative homologs of Type 2 rhodopsin in several species of basal fungi is provided in Chapter~\ref{chap:RhodStruct}.\\
\indent Biosynthesis of $\beta$-carotene begins with phytoene cyclase
converting two molecules of geranylgeranyl pyrophosphate to one
molecule of phytoene. Phytoene desaturase then acts in a five-step pathway to convert phytoene into lycopene \cite{Hausmann2000}. Lycopene cyclase finally acts to convert lycopene to $\beta$-carotene \cite{Cunningham1994}. Subsequently, two different cleavage enzymes can potentially act on $\beta$-carotene. The enzyme $\beta$,$\beta$-carotene 15,15'-monooxygenase 1 (BCMO1) cleaves $\beta$-carotene into two all-\textit{trans}-retinal molecules, and is considered a key enzyme for retinoid metabolism \cite{Lietz2012}. The structurally related enzyme $\beta$,$\beta$-carotene 9',10'-dioxygenase (BCDO2) also acts on $\beta$-carotene to produce $\beta$-apo-10'-carotenal and $\beta$-ionone, however its physiological role is less well-characterized \cite{Lobo2012}. Comparative analysis and discussion of retinal biosynthesis enzymes is provided in Chapter~\ref{chap:ClatTranscriptome}.\\
\subsection*{Towards the development of bioinformatic resources for entomopathogenic Blastocladiomycete \textit{Coelomomyces lativittatus}}
While fungi which invade insects have been observed since antiquity 
(by some estimates, approximately 900 AD in Japan), the exact nature 
of the fungal-insect relationship was not cleanly determined until 
around the 1880s \cite{Samson1988}. After this relationship was established, 
research during the following several decades primarily focused on 
development of applications related to control of agricultural pests. During 
the mid-20th century, interest in these sorts of applied pest control strategies 
waned, but taxonomic knowledge greatly increased. In the late 20th century renewed 
interest in alternative pest control strategies picked up, and due to enhanced 
technology (microbial, genetic, genomic techniques) this is ongoing \cite{Samson1988}.\\
\indent In basal lineages, members of the genus \textit{Coelomomyces} in the
Blastocladiomycota are the only known chytrid entomopathogens. There are
more than 70 described species of \textit{Coelomomyces}, though the true
diversity is estimated be be several hundred \cite{Couch1985}. While
initially studied as a promising avenue for mosquito control, and an
alternative to traditional pesticides, difficulties in culturing
\textit{Cl} have lead to a decline in its research. However its
specific host range and continued search for pesticide alternatives
have allowed it to persist as an interesting avenue of research.\\
\indent Chapter~\ref{chap:ClatTranscriptome} presents a preliminary analysis of transcriptome data obtained from \textit{Coelomomyces lativittatus}. This analysis serves two purposes. First, it lays the groundwork for future RNASeq and proteomic studies of this organism. And secondarily, it attempts to assign molecular detail to previously published observational research about certain biochemical mechanisms (eg $\beta$-carotene production and photoreception).\\
\subsection*{Eukaryotic Flagellar motility}
One of the defining characteristics of the early-diverging fungal
lineages is the presence of a posterior flagellum, which is used by
the zoospores for motility \cite{Koch1958motileI}. The chytrid flagellar
apparatus is composed of the flagellar stalk (axoneme), the kinetosome
(basal body), and the rootlet system \cite{Barr1981}. Microscopy analyses from Koch 
and others \cite{Koch1958motileI} describe nine fibril doublets surrounding a paired central core, which is characteristic of the
"9+2" arrangement of microtubule stalks found in other eukaryotic flagella, such 
as those of human and \textit{Ciona intestinalis} sperm cells \cite{Inaba2003}, 
and \textit{Chlamydomonas reinhardtii} \cite{Siflow2001}. The use of these morphological 
characteristics as phylogenetic markers is supported by modern nucleic 
acid-based phylogenetic techniques \cite{James2000}.\\
\indent During the course of fungal evolution, there was a transition from flagellated motile aquatic single celled organisms to terrestrial multicellular organisms \cite{Taylor2006,Stajich2009}. There is support for anywhere from a single flagellar loss event \cite{Liu2006} to at least four different such events \cite{James2006sixGene} prior to the divergence of the Zygomycota.\\
\indent The chytrid flagellum is the primary method of zoospore motility. In most cases, the chytrid flagellum exists as a posteriorly oriented appendage, with a few exceptions. The zoospores of the Neocallimastigomycota lineages, species most commonly found in the anaerobic environment of the mammalian rumen, are posteriorly multiflagellated \cite{Ho1995}. In the Blastocladiomycota, \textit{Coelomomyces} species are biflagellate during a part of their life cycle after the uniflagellate gametes of opposing mating types fuse to form a biflagellate zygote \cite{Padua1986}.\\ 
\indent A comparative genomic study of the chytrid flagellar apparatus is presented in Appendix~\ref{app:Flagella}. Included is a collection of genes which serve as a "core chytrid" flagellar geneset, which may prove useful in future assessments of chytrid and other basal fungal genomes.\\
