%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Document: Thesis for PhD at UC Riverside                          %%
%% Description: A comparative analysis of environment sensing in EDF %%
%% Author: Steven Ahrendt                                            %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% MISC LIST OF DATA %
%%%%%%%%%%%%%%%%%%%%%

% Taxa and sources
<<label=AppData_taxa,echo=FALSE,results=tex>>=
library(xtable)
data <- read.table("../dat/taxonlistID",sep="\t",row.names=1,header=F,check.names=FALSE,comment.char="#",stringsAsFactors=FALSE)
colnames(data) <- c("Key","Group","TaxID","Name","Isolate/Version","Source")
data$Name <- paste("\\emph{",data$Name,"}",sep="")
xdata <- xtable(data,label="tab:AppData_taxa",caption=c('List of all proteomes used in comparative analyses, including NCBI taxonomy database ID, phylogenetic abbreviation ("Key"), phylum designation ("Group"), isolate number/version (where applicable), and source website.','List of proteomes used in comparative analyses.'))
hlines <- c(-1,-1,0,nrow(xdata),nrow(xdata))
print(xdata,
      table.placement="tbp",
      caption.placement="top",
      hline.after=hlines)
@

% PDB IDs
<<label=AppData_PDB,echo=FALSE,results=tex>>=
library(xtable)
data <- read.table("../dat/PDBIDs.tsv",sep="\t",header=T,check.names=FALSE)
xdata <- xtable(data,label="tab:AppData_PDB",caption=c("PDB IDs and descriptions of structures used in docking and Molecular dynamics simulations","PDB IDs and descriptions of structures used in comparative analyses"))
hlines <- c(-1,-1,0,nrow(xdata),nrow(xdata))
print(xdata,table.placement="tbp",caption.placement="bottom",hline.after=hlines)
@
