%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Document: Thesis for PhD at UC Riverside                                    %%
%% Title: Investigating the evolution of environmental and biotic interactions %%
%%          in basal fungal lineages through comparative genomics              %%
%% Author: Steven Ahrendt                                                      %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
% COELOMOMYCES TABLES %
%%%%%%%%%%%%%%%%%%%%%%%

% latex table generated in R 3.0.1 by xtable 1.7-4 package
% Wed Jul  1 22:39:05 2015
\begin{table}[hp]
\caption[Top 20 PFAM domain counts]{Comparisions of counts of top 20 PFAM domains from \textit{C. lativittatus} as identified in other chytrids. Details for organism abbreviations can be found in appendix.} 
\label{tab:ChClat_PFAM}
\begin{tabular}{rlrrrrrr}
  \hline
\hline
 & Description & \emph{Clat} & \emph{Amac} & \emph{Cang} & \emph{Spun} & \emph{Bden} & \emph{Hpol} \\ 
  \hline
PF00069 & Pkinase & 297 & 409 & 185 & 354 & 404 & 149 \\ 
  PF00400 & WD40 & 211 & 1287 & 185 & 1710 & 1075 & 170 \\ 
  PF00076 & RRM 1 & 143 & 201 &  94 & 278 & 240 &  53 \\ 
  PF00153 & Mito carr & 119 & 222 &  42 & 294 & 231 &  49 \\ 
  PF00012 & HSP70 & 112 &  42 &  11 &  18 &  16 &   9 \\ 
  PF00118 & Cpn60 TCP1 &  93 &  22 &  11 &  20 &  31 &  10 \\ 
  PF00071 & Ras &  86 &  80 &  57 & 108 &  96 &  47 \\ 
  PF00270 & DEAD &  70 & 115 &  94 & 136 & 126 &  81 \\ 
  PF00036 & efhand &  67 &  27 &  49 &  60 &  68 &  30 \\ 
  PF00112 & Peptidase C1 &  65 &   0 &   1 &   0 &   0 &   0 \\ 
  PF01576 & Myosin tail 1 &  59 &   1 &  42 &   4 &   3 &  10 \\ 
  PF00004 & AAA &  58 & 128 & 121 & 134 & 135 &  93 \\ 
  PF00271 & Helicase C &  52 & 140 &  77 & 184 & 199 &  71 \\ 
  PF00227 & Proteasome &  52 &  26 &  16 &  28 &  43 &  19 \\ 
  PF01532 & Glyco hydro 47 &  52 &   2 &   3 &   8 &  12 &   8 \\ 
  PF07690 & MFS 1 &  50 & 144 &  57 & 120 &  69 &  36 \\ 
  PF02985 & HEAT &  50 &  34 &  64 &  56 & 136 &  58 \\ 
  PF00005 & ABC tran &  50 & 217 & 180 & 168 & 231 &  96 \\ 
  PF00009 & GTP EFTU &  48 &  79 &  80 &  92 &  48 &  54 \\ 
  PF00089 & Trypsin &  47 &   2 &   9 &   8 &   4 &   2 \\ 
   \hline
\hline
\end{tabular}
\end{table}
% latex table generated in R 3.0.1 by xtable 1.7-4 package
% Wed Jul  1 22:39:05 2015
\begin{landscape}
\begin{table}[hp]
\caption[BLASTP results for \textit{B. emersonii} $\beta$-carotene genes]{Top BLASTP hits against SwissProt for three \textit{B. emersonii} $\beta$-carotene metabolism genes. PMID references publications for SwissProt hit describing experimental verification of biological function} 
\label{tab:ChClat_BemeVerify}
\begin{tabular}{rllllllrrrr}
  \hline
\hline
  EMBLID & UniprotID & \emph{B. emersonii} Description & TopHitID & Top hit Description & Source organism & \% Cov & Eval & \% ID & PMID \\ 
  \hline
  KJ468786 & A0A060GS52 & putative phytoene dehydrogenase & P54982 & phytoene desaturase & \emph{Phycomyces blakesleeanus} NRRL 1555 &  85 & 0.00 &  53 & 9079885 \\ 
  KJ468785 & A0A060GVE0 & putative lycopene cyclase / phytoene synthase & Q9UUQ6 & bifunctional lycopene cyclase/phytoene synthase & \emph{Mucor circinelloides f. lusitanicus} &  93 & 0.00 &  33 & 10951210 \\ 
  KJ468787 & A0A060GW07 & putative carotenoid dioxygenase & Q9I993 & Beta,beta-carotene 15,15'-monooxygenase & \emph{Gallus gallus} &  51 & 0.00 &  27 & 10799297 \\ 
   \hline
\hline
\end{tabular}
\end{table}
\end{landscape}

% latex table generated in R 3.0.1 by xtable 1.7-4 package
% Wed Jul  1 22:39:05 2015
\begin{table}[hp]
\caption[$\beta$-carotene HMM]{Sequences from KEGG used in generating HMMs for $\beta$-carotene metabolism searches} 
\label{tab:ChClat_KEGGHMM}
\begin{tabular}{rlll}
  \hline
\hline
  KEGG.ID & Description & Sequences used \\ 
  \hline
  K10027 & phytoene desaturase & 442 Bacterial proteins, 49 Archaeal proteins \\ 
  K02291 & phytoene synthase & 6 Eukaryote proteins, 45 Plant proteins [excluded 49 Archaeal and 843 Bacterial proteins] \\ 
  K00515 & beta-carotene 15,15'-monooxygenase & 64 Metazoan proteins [excluded 1 Archaeal protein] \\ 
   \hline
\hline
\end{tabular}
\end{table}
% latex table generated in R 3.0.1 by xtable 1.7-4 package
% Wed Jul  1 22:39:05 2015
\begin{table}[hp]
\caption[orthoMCL clusters for transmembrane proteins]{Unique clusters of proteins with 7-9 membrane-spanning domain, grouped using OrthoMCL} 
\label{tab:ChClat_orthomcl}
\begin{tabular}{rrll}
  \hline
\hline
  Cluster & Protein & Description \\ 
  \hline
  1127 & Clat$|$m.15402 & 7tm\_1;PF00001 \\ 
  1127 & Clat$|$m.16182 & 7tm\_1;PF00001 \\ 
  1127 & Clat$|$m.18794 & 7tm\_1;PF00001 \\ 
  1127 & Clat$|$m.9338 & 7tm\_1;PF00001 \\ 
  1175 & Clat$|$m.12314 & K\_trans;PF02705 \\ 
  1175 & Clat$|$m.12319 & K\_trans;PF02705 \\ 
  1175 & Clat$|$m.12322 & K\_trans;PF02705 \\ 
  1176 & Clat$|$m.15582 & Grp1\_Fun34\_YaaH;PF01184 \\ 
  1176 & Clat$|$m.15583 & Grp1\_Fun34\_YaaH;PF01184 \\ 
  1176 & Clat$|$m.15584 & Grp1\_Fun34\_YaaH;PF01184 \\ 
  1177 & Clat$|$m.16070 & DUF3533;PF12051 \\ 
  1177 & Clat$|$m.16072 & DUF3533;PF12051 \\ 
  1177 & Clat$|$m.16075 & DUF3533;PF12051 \\ 
  1269 & Clat$|$m.11119 & UAA transporter;PF08449 \\ 
  1269 & Clat$|$m.11121 & UAA transporter;PF08449 \\ 
  1270 & Clat$|$m.12151 & -- \\ 
  1270 & Clat$|$m.12173 & -- \\ 
  1271 & Clat$|$m.8230 & -- \\ 
  1271 & Clat$|$m.8232 & -- \\ 
  1272 & Clat$|$m.12638 & -- \\ 
  1272 & Clat$|$m.12641 & -- \\ 
  1273 & Clat$|$m.14468 & Sodium:sulfate symporter;PF00939 \\ 
  1273 & Clat$|$m.14469 & Sodium:sulfate symporter;PF00939 \\ 
  1274 & Clat$|$m.14625 & -- \\ 
  1274 & Clat$|$m.14626 & -- \\ 
  1275 & Clat$|$m.4725 & NADH-Ubiquinone/plastoquinone;PF00361 \\ 
  1275 & Clat$|$m.8681 & NADH-Ubiquinone/plastoquinone;PF00361 \\ 
  1276 & Clat$|$m.4968 & Chitin synthase;PF03142 \\ 
  1276 & Clat$|$m.4970 & Chitin synthase;PF03142 \\ 
   \hline
\hline
\end{tabular}
\end{table}

% latex table generated in R 3.0.1 by xtable 1.7-4 package
% Wed Jul  1 22:39:05 2015
\begin{table}[hp]
\caption[\textit{C. lativittatus} photosensing proteins]{Predicted \textit{C. lativittatus} proteins associated with photosensing. PFAM Family definitions: PF13426, "PAS domain" [PAS\_9]; PF08447, "PAS fold" [PAS\_3]; PF00001, "7 transmembrane receptor (rhodopsin family)" [7tm\_1]; PF01036, "bacteriorhodopsin-like protein" [Bac\_rhodopsin]; PF10317, "Serpentine type 7TM GPCR chemoreceptor Srd" [7TM\_GPCR\_Srd]; PF00211, "Adenylate and Guanylate cyclase catalytic domain" [Guanylate\_cyc]} 
\label{tab:ChClat_photosensing}
\begin{tabular}{rlllllr}
  \hline
\hline
  OrfID & Type & UniProtKB.BLAST.match & InterPro & TMPred & FPKM \\ 
  \hline
  m.12730 & WC-1 & \emph{Neurospora crassa}\verb|^|Q01371\verb|^|4e-43 & -- & -- & 2.37 \\ 
  m.7819 & OpGC & \emph{Halobacterium sp.} NRC-1\verb|^|P71411\verb|^|7.4e-07;Drosophila melanogaster\verb|^|Q8INF0\verb|^|8e-44 & PF01036\verb|^|32:241\verb|^|5.0e-37;PF00211\verb|^|355:533\verb|^|4.6e-51 & -- & 18.00 \\ 
  m.9338 & Op & \emph{Danio rerio}\verb|^|Q2KNE5\verb|^|3e-08 & PF10317\verb|^|21:205\verb|^|1.9e-7;PF00001\verb|^|50:355\verb|^|1.5e-17 & -- & 12.40 \\ 
  m.15402 & Op & \emph{Apis mellifera}\verb|^|Q17053\verb|^|8e-10 & PF00001\verb|^|36:341\verb|^|1.0e-21 & -- & 5.17 \\ 
  m.16182 & Op & \emph{Cambarus hubrichti}\verb|^|O18312\verb|^|1e-04 & PF00001\verb|^|14:255\verb|^|9.7e-13 & -- & 4.70 \\ 
  m.18794 & Op & \emph{Limulus polyphemus}\verb|^|P35360\verb|^|3e-10 & PF00001\verb|^|20:323\verb|^|5.9e-22 & -- & 1.62 \\ 
  m.11198 & Op & \emph{Halobacterium sp.} AUS-2\verb|^|P29563\verb|^|3e-30 & PF01036\verb|^|32:241\verb|^|5.0e-37 & -- & 1.54 \\ 
   \hline
\hline
\end{tabular}
\end{table}
% latex table generated in R 3.0.1 by xtable 1.7-4 package
% Wed Jul  1 22:39:05 2015
\begin{table}[hp]
\caption[FAADB predictions for chytrid proteins]{Fungal Adhesin and Adhesin-like Database (FAADB) predictions for chytrid proteins. Descriptions for chytrid proteins identified using positive dataset from predicted adhesin and adhesin-like proteins.} 
\label{tab:ChClat_FAADB}
\begin{tabular}{rrllrrr}
  \hline
\hline
 & SVM.Score & UniprotKB.ID & Description & Identity & Alignment.Length & E.value \\ 
  \hline
Amac$|$AMAG\_17113T0 & -0.74 & Q9DBG3.1 & AP-2 complex subunit beta & 55.26 & 903 & 0.00 \\ 
  Amac$|$AMAG\_05862T0 & -0.61 & P00940.2 & Triosephosphate isomerase EC=5.3.1.1 & 58.06 & 248 & 0.00 \\ 
  Amac$|$AMAG\_12167T0 & -0.54 & P00940.2 & Triosephosphate isomerase EC=5.3.1.1 & 59.27 & 248 & 0.00 \\ 
  Amac$|$AMAG\_08758T0 & -0.19 & P46226.3 & Triosephosphate isomerase, cytosolic EC=5.3.1.1 & 59.84 & 249 & 0.00 \\ 
  Amac$|$AMAG\_10182T0 & -0.17 & P46226.3 & Triosephosphate isomerase, cytosolic EC=5.3.1.1 & 60.24 & 249 & 0.00 \\ 
  Amac$|$AMAG\_18542T0 & -0.16 & P48491.2 & Triosephosphate isomerase, cytosolic EC=5.3.1.1 & 58.19 & 177 & 0.00 \\ 
  Amac$|$AMAG\_16371T0 & 0.15 &  &  &  &  &  \\ 
  Amac$|$AMAG\_08832T0 & 0.17 & Q966L9.1 & ATP-dependent RNA helicase glh-2 EC=3.6.4.13 & 54.63 & 108 & 0.00 \\ 
  Amac$|$AMAG\_17977T0 & 0.21 &  &  &  &  &  \\ 
  Amac$|$AMAG\_16084T0 & 0.32 & P29141.1 & Minor extracellular protease vpr EC=3.4.21.- & 42.00 & 600 & 0.00 \\ 
  Amac$|$AMAG\_04496T0 & 0.34 & Q6H236.1 & Paternally-expressed gene 3 protein & 45.71 & 525 & 0.00 \\ 
  Amac$|$AMAG\_19066T0 & 0.76 &  &  &  &  &  \\ 
  Amac$|$AMAG\_20477T0 & 0.83 &  &  &  &  &  \\ 
  Amac$|$AMAG\_04749T0 & 1.33 &  &  &  &  &  \\ 
  Amac$|$AMAG\_03430T0 & 1.43 &  &  &  &  &  \\ 
  Cang$|$CANG\_48379 & -0.78 & Q06852.2 & Cell surface glycoprotein 1 & 29.47 & 431 & 0.00 \\ 
  Cang$|$CANG\_125451 & -0.68 & P58559.1 & Glyceraldehyde-3-phosphate dehydrogenase 3 EC=1.2.1.12 & 58.02 & 343 & 0.00 \\ 
  Cang$|$CANG\_33361 & -0.09 & P48494.3 & Triosephosphate isomerase, cytosolic EC=5.3.1.1 & 59.04 & 249 & 0.00 \\ 
  Cang$|$CANG\_38430 & 0.60 & P22105.3 & Tenascin-X & 39.67 & 673 & 0.00 \\ 
  Cang$|$CANG\_69396 & 0.94 &  &  &  &  &  \\ 
  Clat$|$m.10957 & -0.78 & P27393.1 & Collagen alpha-2(IV) chain & 63.61 & 665 & 0.00 \\ 
  Clat$|$m.22466 & -0.75 & Q6CJG5.2 & Triosephosphate isomerase EC=5.3.1.1 & 48.60 & 179 & 0.00 \\ 
  Clat$|$m.2165 & -0.68 & A7S7F2.1 & Bystin & 43.48 & 276 & 0.00 \\ 
  Clat$|$m.18806 & -0.68 & P48501.1 & Triosephosphate isomerase EC=5.3.1.1 & 56.19 & 226 & 0.00 \\ 
  Clat$|$m.13634 & -0.63 & Q90XG0.1 & Triosephosphate isomerase B EC=5.3.1.1 & 71.02 & 245 & 0.00 \\ 
  Clat$|$m.1572 & -0.58 & O09452.1 & Glyceraldehyde-3-phosphate dehydrogenase, chloroplastic EC=1.2.1.59 & 84.01 & 269 & 0.00 \\ 
  Clat$|$m.15929 & -0.52 & P07487.2 & Glyceraldehyde-3-phosphate dehydrogenase 2 EC=1.2.1.12 & 80.28 & 142 & 0.00 \\ 
  Clat$|$m.10361 & -0.46 & P20445.2 & Glyceraldehyde-3-phosphate dehydrogenase EC=1.2.1.12 & 61.34 & 238 & 0.00 \\ 
  Clat$|$m.18916 & -0.42 & P48501.1 & Triosephosphate isomerase EC=5.3.1.1 & 60.87 & 253 & 0.00 \\ 
  Clat$|$m.11233 & -0.30 & Q96UF2.1 & Glyceraldehyde-3-phosphate dehydrogenase 2 EC=1.2.1.12 & 68.82 & 279 & 0.00 \\ 
  Clat$|$m.4480 & -0.07 & O77458.1 & Triosephosphate isomerase EC=5.3.1.1 & 64.24 & 165 & 0.00 \\ 
  Clat$|$m.13062 & 0.52 & Q6BMK0.1 & Glyceraldehyde-3-phosphate dehydrogenase EC=1.2.1.12 & 74.52 & 157 & 0.00 \\ 
  Clat$|$m.745 & 0.90 & Q92824.4 & Proprotein convertase subtilisin/kexin type 5 EC=3.4.21.- & 36.65 & 562 & 0.00 \\ 
  Clat$|$m.16183 & 1.05 & P22105.3 & Tenascin-X & 51.01 & 545 & 0.00 \\ 
  Clat$|$m.13342 & 1.13 & Q92824.4 & Proprotein convertase subtilisin/kexin type 5 EC=3.4.21.- & 43.22 & 1025 & 0.00 \\ 
  Clat$|$m.11209 & 2.24 &  &  &  &  &  \\ 
  Bden$|$BDET\_06684 & -0.79 & Q5R2J2.1 & Glyceraldehyde-3-phosphate dehydrogenase EC=1.2.1.12 EC=2.6.99.- & 71.56 & 334 & 0.00 \\ 
  Bden$|$BDET\_05736 & -0.17 & P00939.1 & Triosephosphate isomerase EC=5.3.1.1 & 57.66 & 248 & 0.00 \\ 
  Bden$|$BDET\_05372 & -0.03 & Q95P23.1 & Enterin neuropeptides & 28.02 & 182 & 0.00 \\ 
  Bden$|$BDET\_01761 & 0.45 & Q9AVB0.1 & Lectin-B & 64.57 & 460 & 0.00 \\ 
  Bden$|$BDET\_00626 & 0.49 &  &  &  &  &  \\ 
  Bden$|$BDET\_00287 & 0.67 & Q9AVB0.1 & Lectin-B & 58.01 & 462 & 0.00 \\ 
  Bden$|$BDET\_02239 & 1.45 & P23253.1 & Sialidase EC=3.2.1.18 & 54.09 & 318 & 0.00 \\ 
  Bden$|$BDET\_00436 & 1.48 &  &  &  &  &  \\ 
  Bden$|$BDET\_03668 & 1.51 & Q63425.2 & Periaxin & 39.51 & 367 & 0.00 \\ 
  Bden$|$BDET\_06100 & 1.70 &  &  &  &  &  \\ 
  Hpol$|$HPOL\_4940 & -0.79 & P00940.2 & Triosephosphate isomerase EC=5.3.1.1 & 56.63 & 249 & 0.00 \\ 
  Hpol$|$HPOL\_4790 & -0.46 & O57479.3 & Glyceraldehyde-3-phosphate dehydrogenase EC=1.2.1.12 EC=2.6.99.- & 75.61 & 328 & 0.00 \\ 
  Hpol$|$HPOL\_4769 & -0.24 & P52041.2 & 3-hydroxybutyryl-CoA dehydrogenase EC=1.1.1.157 & 40.00 &  80 & 0.00 \\ 
  Hpol$|$HPOL\_2513 & -0.14 & Q1ZXD6.1 & Probable serine/threonine-protein kinase roco5 EC=2.7.11.1 & 32.78 & 180 & 0.00 \\ 
  Spun$|$SPPG\_01012T0 & -0.68 & P30741.2 & Triosephosphate isomerase EC=5.3.1.1 & 61.60 & 250 & 0.00 \\ 
  Spun$|$SPPG\_08522T0 & -0.47 & Q6ZRI0.3 & Otogelin & 31.26 & 531 & 0.00 \\ 
  Spun$|$SPPG\_00685T0 & -0.15 & Q5H8C1.3 & FRAS1-related extracellular matrix protein 1 & 33.98 & 1292 & 0.00 \\ 
  Spun$|$SPPG\_01988T0 & 0.05 &  &  &  &  &  \\ 
   \hline
\hline
\end{tabular}
\end{table}
% latex table generated in R 3.0.1 by xtable 1.7-4 package
% Wed Jul  1 22:39:05 2015
\begin{table}[hp]
\caption[BLASTP hits for \textit{C. lat} m.10080]{BLASTP results using \textit{C. lativittatus} ORF m.10080 as a query against SwissProt database. Top five hits are to liver X receptors (LXRs) from various mammalian species. The next siz hits are only those which were to 20-hydroxy-ecdysone receptors, all of which are from insects} 
\label{tab:ChClat_LBD}
\begin{tabular}{rlrll}
  \hline
\hline
 & Coverage & Eval & \% Identity & Description \\ 
  \hline
Q5E9B6.1 & 40\% & 0.00 & 28\% & Liver X receptor alpha [\emph{Bos taurus}] \\ 
  Q13133.2 & 40\% & 0.00 & 28\% & Liver X receptor alpha [\emph{Homo sapiens}] \\ 
  Q62685.1 & 40\% & 0.00 & 28\% & Liver X receptor alpha [\emph{Rattus norvegicus}] \\ 
  Q9Z0Y9.3 & 40\% & 0.00 & 28\% & Liver X receptor alpha [\emph{Mus musculus}] \\ 
  P55055.2 & 40\% & 0.00 & 28\% & Liver X receptor beta [\emph{Homo sapiens}] \\ 
   \hline
P49880.2 & 36\% & 0.00 & 28\% & 20-hydroxy-ecdysone receptor [\emph{Aedes aegypti}] \\ 
  P49883.1 & 42\% & 0.00 & 24\% & 20-hydroxy-ecdysone receptor [\emph{Manduca sexta}] \\ 
  P49881.1 & 42\% & 0.00 & 24\% & 20-hydroxy-ecdysone receptor [\emph{Bombyx mori}] \\ 
  P34021.1 & 36\% & 0.00 & 26\% & 20-hydroxy-ecdysone receptor [\emph{Drosophila melongaster}] \\ 
  O18531.2 & 36\% & 0.00 & 27\% & 20-hydroxy-ecdysone receptor [\emph{Lucilia cuprina}] \\ 
  P49882.1 & 36\% & 0.00 & 25\% & 20-hydroxy-ecdysone receptor [\emph{Chironomus tentans}] \\ 
   \hline
\hline
\end{tabular}
\end{table}
