%%%%%%%%%%%%%%%%%
%% Document: Thesis for PhD at UC Riverside
%% Description: Comparative genomics analysis in early-diverging fungi
%% Author: Steven Ahrendt
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Transcriptome analysis in Coelomomcyes lativittatus %
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Transcriptome analysis in \textit{Coelomomyces lativittatus}}
\label{chap:Clat_transcriptome}
\section{Introduction}
\subsection{Background of chytrids and Coelomomyces and life cycle}
Species of Coelomomyces (Blastocladiomycota; Blastocladiales; Coelomomycetaceae) are obligate parasites which cycle between insect and crustacean hosts (Whisler et al. 1975). This process begins when biflagellate zygotes encounter mosquito larvae. The spore settles on and attaches to the host cuticle, a process facilitated by the secretion of adhesion vesicles which contain a glue-like substance (Travland 1979a). After secretion of a thin cell wall, the encysted spore develops an appressorium and penetration tube which breaks through the host cuticle (Zebold et al. 1979). Once inside the larval hemocoel, the spore develops into a sporangia. Host death liberates these sporangia. Meiosis within the sporangia produces haploid uniflagellate meiospores of opposing mating types, which are subsequently released to individually infect crustacean hosts (typically copepods, though ostracods can serve as hosts as well (Whisler et al. 2009)). The penetration of copepods is thought to occur in a manner similar to that of the mosquito larvae (Zebold et al. 1979). Gametangia develop from these meiospores within the copepod hemocoel, which are ultimately cleaved into gametes and released upon crustacean host death. In the environment once again, opposing gametes fuse to create biflagellate zygotes, which propagate the cycle by infecting new mosquito larvae (Whisler et al. 1975).\\
\indent \textit{C. lativittatus} is a member of the Blastocladiomycota, one of the zoosporic fungal lineages. This polyphyletic group, commonly referred to as “chytrids”, comprises the Chytridiomycota, Blastocladiomycota, Neocallimastigomycota, and Cryptomycota. These lineages are sister to the Dikarya and Zygomycete lineages, and diverged close to the initial fungal-metazoan divergence. One of the major characteristics of all chytrid fungi is an asexual life cycle featuring a posteriorly flagellate zoospore, which is a motile cell capable of exploring its environment for a suitable substrate upon which to encyst and propagate. These lineages contain species with a wide variety of nutritional modes, and substrates utilized by various chytrid species include decaying plant matter (eg. \textit{Spizellomyces punctatus} (Barr 1984)), living plant matter (eg. \textit{Olpidium brassicae} (Tewari and Bains 1983) and \textit{Rhizophidium graminis} (Macfarlane 1970)), amphibian skin (eg. \textit{Batrachochytrium dendrobatidis} (Longcore et al. 1999)), and even other chytrids (\textit{Rozella allomyces}, an endosymbiont of \textit{Allomyces reticulatus} (James et al. 2013)). Despite holding these important ecological positions, chytrids are generally understudied and represent less than two percent of described fungi (Stajich et al. 2009). This study is motivated by a desire to understand Coelomomyces specifically and add to the growing knowledge of chytrid biology in general. \\
\subsection{Insect virulence}
Coelomomyces species have been studied previously in the context of mosquito control (Scholte et al. 2004). While the potential for use as a biological control agent has been explored, the exact biochemical nature of mosquito infection, including descriptions of all enzymes and pathways involved, has not. \\
\indent There are many examples of entomopathogenic organisms specializing in mosquito hosts, covering 13 genera across 2 kingdoms (Fungi and Chromista) (Scholte et al. 2004). The Ascomycete fungus Metarhizium anisopliae is one of the well-studied fungal models for investigations into this specialized group. Early research looked at the range of enzymes produced by pathogenic isolates of this fungus, and identified a variety including proteases, amino-/carboxy- peptidases, lipases, esterases, chitinases, NAGases, catalases, polyphenol oxidases, and deoxy- and ribonucleases (St. Leger et al. 1986). Later studies added to this repertoire the production of toxic cyclic peptides known as destruxins (Wang et al. 2012). \\ 
\subsection{Sensory capacity (What are some types of sensory receptors)}
The dual-host, multistage life cycle of Coelomomyces, which passes through a number of chemically distinct environments, suggests the presence of an elaborate sensory repertoire. For instance, experimental evidence demonstrates that gametes of some Coelomomyces species are specifically attracted to mosquito ovaries, and that this attraction is, at least in part, mediated by the hormone 20-hydroxyecdysone (Lucarotti 1992). Other evidence demonstrates a species-specific, photoperiod-dependent periodicity of gamete release from the copepod host (Federici 1983), strongly implying that Coelomomyces has the molecular capacity for some manner of circadian rhythm regulation. \\
\indent There are many classes of photoreceptor proteins in fungi capable of producing a cellular response from an environmental light signal, all of which have different mechanisms of action and specializations: phytochromes, cryptochromes, the white-collar complex, and opsins (Idnurm et al. 2010). In plants, phytochromes function as day-night sensors to regulate the circadian rhythm and flowering response. This is accomplished through a conformational shift between the red and far-red sensitive forms of the protein structure (Rockwell et al. 2006). While relatively little is known about fungal phytochrome function, research on A. nidulans suggests that the phytochrome protein is a member of an elaborate complex with regulatory functions involved with the asexual-sexual transition and secondary metabolite biosynthesis (Idnurm et al. 2010). \\
\indent Cryptochromes, found predominantly in plants, animals, and insects, are blue-light sensitive proteins involved in circadian rhythm regulation and light activated DNA damage repair (Idnurm et al. 2010). Additional evidence suggests cryptochrome proteins play a role in mediating the phototactic behavior of sponge larvae (Rivera et al. 2012). \\
\indent First studied in the model filamentous Ascomycete fungus Neurospora crassa, the well-characterized white-collar complex assembles as a heterodimer comprising White-collar 1 and 2 proteins. This complex functions to sense blue and near UV wavelengths, and, when active, directly interacts with DNA to regulate the circadian clock machinery, sporulation, pigmentation, and phototropism (Ballario and Macino 1997; Purschwitz et al. 2006; Corrochano 2007). \\ 
\indent The largest family of membrane receptors by far is that of the seven-transmembrane (7TM) receptors, comprising upwards of 800 genes (Pierce et al. 2002). This family includes various receptors for a wide range of ligands, including hormones, neurotransmitters, odorants, and photons. While there are three distinct subfamilies (A, B, and C), they share very little sequence similarity. Opsins, examples of which can be found in bacteria, archaea, and eukaryotes, are part of the largest family of 7TM proteins. These G-protein-coupled receptors (GPCRs) function via photoisomerization of a covalently bound retinyledene chromophore, typically 11-cis-retinal (Shichida and Matsuyama 2009). This and other isomers of retinal are biosynthesized from the red-orange organic compound β-carotene. \\
 Coelomomyces are known producers of β-carotene (Federici and Thompson 1979), the production of which is indicative of mating type, resulting in gametangia and gametes that are either strong orange (arbitrarily "male") or colorless/amber (arbitrarily "female"). β-carotene is ubiquitous in nature and exists primarily as a precursor for the biosynthesis of Vitamin A. Biosynthesis of β-carotene begins with phytoene cyclase converting two molecules of geranylgeranyl pyrophosphate to one molecule of phytoene. Phytoene desaturase then works in a five-step pathway to convert phytoene into lycopene (Hausmann and Sandmann 2000). Lycopene cyclase finally acts to convert lycopene to β-carotene (Cunningham et al. 1994). Subsequently, two different cleavage enzymes can potentially act on β-carotene. The enzyme β,β-carotene 15,15'-monooxygenase 1 (BCMO1) cleaves β-carotene into two all-trans-retinal molecules, and is considered a key enzyme for retinoid metabolism (Lietz et al. 2012). The structurally related enzyme β,β-carotene 9',10'-dioxygenase (BCDO2) also acts on β-carotene to produce β-apo-10'-carotenal and β-ionone, however its physiological role is less well-characterized (Lobo et al. 2012).
\subsection{Bigger picture}
The total number of species of Coelomomyces worldwide is estimated to be several hundred, yet little is known about the more detailed aspects of biochemistry and genomics. Therefore, an ongoing effort toward the assembly and annotation of a Coelomomyces transcriptome will not only add to the growing collection of knowledge about chytrid fungi broadly, but will also provide new insights into the underlying mechanisms that govern the alternating life cycle of Coelomomyces and can help further its development as a biological agent of mosquito control. \\
\indent This research represents the first exploratory investigation of Coelomomyces genomics using the transcriptome of Coelomomyces lattivitatus. Herein, we compare expressed protein functions relative to other zoosporic fungi, biochemically reconstruct known pathways of carotenoid and retinal biosynthesis, and identify potential members of what is presumed to be a vast and complicated sensory network. \\
%% Results
%%%%%%%%%%%%%
\section{Results}
\subsection{Transcriptome Characterization}
After quality trimming, we obtained a total of 28,698,279 reads with an average length of 196 nt. De novo assembly of reads using Trinity (Grabherr et al. 2011) yielded 77,597 transcripts with an average length of 386 bp. Within these transcripts, 21,486 open reading frames were predicted using Transdecoder (Haas et al. 2013). Annotation with Trinotate predicted 12,156 transcripts with a BLASTp hit, 11,040 with predicted Pfam domain(s), and 29,076 with associated GO terms. \\
\indent The top 20 PFAM domains identified in the C. lativittatus transcriptome and their respective counts in other chytrids are provided in (Table1\_Clat\_top20\_comparison). The most striking examples of domain families which are underrepresented among other chytrids are trypsin (PF00089), glycoside hydrolase family 47 (PF01532), and papain family cysteine protease (PF00112), all three of which have some manner of protease or carbohydrate degrading functionality. An additional family which appears to be overrepresented in C. lativittatus is the Myosin tail family (PF01576), although the related Blastocladiomycete Cantenaria anguillalae also has a higher number of these proteins relative to other Blastocladiomycete and Chytridiomycete speceis. Corresponding Gene Ontology (GO) Slim classifications for recovered transcripts are shown in (Fig1\_ClatAspergilusGOPlot). \\
\subsection{Insect Virulence} 
At least one gene family appears to be expanded in C. lativittatus, with implications in insect pathogenicity. Also, based on the hypothesis that C. lativittatus senses Anopholes hormones, we predict that there are receptors in the fungus that are similar to or can bind the hormones like 20-hydroxyecdysone. \\
\indent To test for the presence of and possible expansions in gene families that may be related to insect virulence, we scanned the C. lativittatus transcriptome for specific protein domains which have been previously implicated in fungal associated insect virulence, or which may be otherwise related to fungal-insect association. \\
\subsubsection{C1 cysteine proteases} 
The C1 cysteine proteases are commonly found in fruit (eg. papaya) and often used as meat tenderizers. The enzymes from fig, pineapple, and papaya plants have been studied as antihelmintics and found to have high proteolytic activity against nematode cuticles (Stepek et al. 2004). The family is characterized by the Peptidase C1 (PF00112) and C1-like (PF03051) Pfam domains. The C. lativittatus transcriptome contains at least 56 transcripts with peptidase C1 domains (< 98\% identity). Searches of Blastocladiomycete and Chytridiomycetes genomes found no proteins containing these domains, although proteins with this domain are present in the Dikarya lineages. Phylogenetic analysis of the Pfam seed sequences and the fungal copies revealed a number of observations (Fig4\_PF00112). First, the C. lat transcripts are broadly distributed, with very few tight clusters. A group of 5 transcripts fall nicely within a “fungal-specific” lineage containing the only other fungal sequences (derived from the Ascomycota and Basidiomycota). Another cluster of C. lat transcripts cluster as more recent divergences closer to the arthropod sequences. \\
\subsubsection{Trypsin proteases}
Trypsin are serine proteases found in the digestive systems of many vertebrates. These enzymes are characterized by the PF00089 Pfam domain, and 43 transcripts in C. lat were identified. (more here; trees are running)
\subsubsection{Destruxins}
 The destruxins are a class of insecticidal cyclic hexadepsipeptides produced by some entomopathogenic fungi, most notably by species of Metarhizium (Donzelli et al. 2012; Wang et al. 2012). Based on chemical differences in the hydroxy acid, R group, and N-methylation characteristics, these compounds can be divided into a total of 12 chemically distinct classes (Pedras et al. 2002; Wang et al. 2012). The biosynthesis of these compounds is presumed to be mediated by an NRPS gene cluster in Metarhizium robertsii (Wang et al. 2012). A FASTA search with the destruxin synthase (dtxS1) protein in the M. robertsii gene cluster did not identify any putative homologs in our C. lativittatus transcriptome. No putative NRPS or PKS-related proteins searching transcriptome using AntiSmash, though m.15019 (described in β-carotene results as a phytoene synthase) was recovered as a putative terpene synthase. Additionally, no hits for THIOL or CON using HMM searches were recovered (Bushley and Turgeon 2010). Some hits from AMP HMM, but counts are on the order of other chytrids (~15-20).
\subsubsection{Chitin related domains}
Chitin binding domains are a broad class of domains found in carbohydrate-active proteins. Overall, there are 71 different subfamilies within this broad class defined by sequence similarity in the Carbohydrate Active Enzymes database (http://www.cazy.org/; accessed Oct 17, 2014) (Lombard et al. 2014). (presence/absence among chytrids and why that would be important to note here) Five predicted ORFs were identified by InterPro as having a CBM18 domain and six ORFs identified with a CBM33 domain. All of the respective transcripts have minimal gene expression. \\
\indent There was one transcript (m.4968) with a chitin synthase domain annotation (PF03142), and one transcript (m.4725) with a NADH-Ubiquinone domain (PF00361). The FPKM values of these transcripts were 637, and 130, respectively.
\subsubsection{Adhesion-related proteins}
In the infection process, when biflagellate zygotes encounter mosquito larvae, the spore is observed to settle on and attach to the host cuticle. This process is hypothesized to be facilitated by the secretion of so-called “adhesion vesicles” which contain a glue-like substance (Travland 1979). These vesicles have been observed developing prior to the attachment of the spore, localizing to points of contact between the spore and cuticle, and disappearing after host penetration (Travland 1979b). While the chemical nature of these “adhesion vesicles” remains unclear, a number of candidates exist. Fungal adhesins, for example, are membrane proteins which allow certain fungi to attach to surfaces and are usually involved in microbial community biofilm formation. One well studied example is the “hyphal wall protein (Hwp1)” implicated in C. albicans pathogenesis (Staab et al. 1999). However there are no examples of this protein in C. lativittatus or other Blastocladiomycete or Chytridiomycetes surveyed. In an additional attempt to ascertain the nature of spore-cuticle attachment, we probed the Fungal Adhesin and Adhesin-like Database (FaaDB, http://bioinfo.icgeb.res.in/faap/faap.html), a set of experimentally verified and well-annotated fungal adhesins from several different fungi, predominantly Dikarya. The positive dataset was searched against the C. lativittatus transcriptome and well-scoring hits were recovered and submitted to the FAApred SVM-based prediction method trained on both positive and negative adhesin datasets. In C. lativittatus, this method identified 16 sequences were as putative adhesins. In the other Blastocladiomycte and Chytridiomycetes surveyed, 10, 5, 10, 4, and 4 proteins were predicted as such in A. macrogynus, C. anguillulae, B. dendrobatidis, H. polyrhiza, and S. punctatus, respectively. 
\subsubsection{Ecdysone receptors}
The naturally occurring ecdysteroid hormone 20-hydroxyecdysone (20HE) controls moulting in arthropods (Thummel and Chory 2002). There is evidence to suggest that 20HE plays a role in attracting C. stegomyiae to the ovaries of adult female Aedes aegypti (Lucarotti 1992). A FASTA search with the known ecdysone receptor protein from D. melongaster EcR (Koelle et al. 1991) identified a single C. lativittatus transcript. This finding is surprising not only as it provides a straightforward answer to how C. lativittatus could sense its host, but also given the fact that nuclear receptors are not known to be in fungi and are presumed to be only limited to the metazoan lineages (Escriva et al. 1998). An HMM profile constructed from arthropod EcR receptor sequences and human nuclear receptors, when searched against the C. lativittatus transcriptome, identified an additional three transcripts, though the originally identified transcript (m.9546) was still the highest scoring. An alignment is provided in (20HE\_alignment.pdf).\\
\indent This 298 aa transcript is likely not full length, and only aligns to the DNA binding region of the D. mel receptor (approximate residues 239 to 401). The top blast hit for this transcript is the C. elegans nuclear hormone receptor family member nhr-35 (SwissProt accession: Q17771, e-val 2e-20). InterProScan (Jones et al. 2014) predicts the PF00105 domain covering positions 24-92. This domain is a Zinc Finger C4-type and is associated with nuclear receptors. No orthologs of the C. lativittatus transcript were detectable in any other chytrids searching with an e-value threshold of at least 1e-05. \\
\indent Structurally, this transcript is most similar to the DNA-binding region of the D. mel ecdysone receptor (PDB ID: 2HAN, chain B) (Jakób et al. 2007). These two regions have 42\% sequence identity. A homology-based structure model of the C. lativittatus transcript using SwissModel (Arnold et al. 2006) has an RMSD of 0.2 (Dali Server prediction (Holm and Rosenström 2010)) when compared to 2HAN, chain B. \\
\indent The PF00104 ligand binding domain, associated with this and other nuclear receptors in the arthropod receptors, was not predicted to be associated with this transcript. However one other C. lativittatus ORF is predicted to contain the PF00104 domain but has insignificant similarity to the D. mel EcR protein (m.10080, 21.5\% identity, e-val: 0.077). Nonetheless, (TabS1\_LBD\_blastHits) lists BLASTP results after searching m.10080 against SwissProt. The top 5 hits are all to mammalian liver X receptors (LXRs). A total of six hits below a threshold of 1e-06 are to 20HE receptors from insects. All hits have approximately 40\% coverage and approximately 25\% identity. \\
\indent A maximum likelihood tree (Fig\_20HE\_NR\_RAxMLTree) constructed from arthropod 20HE sequences, as well as human nuclear receptor sequences from all nuclear receptor families, shows the C. lativittatus putative DNA-binding homolog sequence clustering outside of the metazoan nuclear receptor sequences. \\
\indent Finally, we wished to determine if any unique receptors are found in C.lat relative to the other, non-insect associated chytrids. Noting that these are phylogenetically very different lineages, we searched for possible receptor candidate genes based on transmembrane domain architecture. In C. lativittatus, 131 transcripts were predicted to have between 6 and 9 transmembrane domains. Of these, 29 are specific and not found in the other Chytridiomycota or Blastocladiomycota genomes surveyed (A. mac, B. den, S. pun, and H. pol) based on ortholog clusters generated with OrthoMCL (Li et al. 2003). These 29 transcripts form 12 unique paralog clusters (meaning groups of only C.lat genes; 29 transcripts distributed among 12 groups). HMMER3 searches of Pfam database identified domains in 8 of these clusters, while the other 4 remained unclassified.\\
\subsection{β-carotene} 
C. lativittatus likely has a typical β-carotene biosynthesis pathway, despite missing enzyme in transcriptome. To determine the molecular characteristics of the β-carotene biosynthesis and metabolism pathways in C. lativittatus, we began by querying the predicted ORFs from the transcriptome with three key enzymes from the biosynthesis pathway described in Blastocladiella emersonii (Avelar et al. 2014). While functional biochemical characterization of these specific B. emersonii enzymes has not been performed, a BLASTP search against the SwissProt database reveals expected top hits with experimental verification of biochemical activity (supTab\_Beme\_verificationOfSequences). \\
\indent The pathway starts with a phytoene desaturase which is necessary for phytoene to lycopene conversion. No candidate C. lativittatus homolog was found with the putative B. emersonii phytoene dehydrogenase sequence (KJ468786) in either the set of predicted ORFs (using the protein sequence in a direct search) nor in the set of assembled transcripts (using the protein sequence in a translated search). Additional queries using phytoene desaturase from Giberella fujikuroi (CarB; UniProt accession: Q8X0Z0) and Neurospora crassa (NCU00552) were similarly unsuccessful. \\
\indent The next enzyme in the process is a lycopene cyclase / phytoene synthase. One transcript, m.15019, contained a 599-aa long predicted ORF, was identified as a putative homolog to B. emersonii bifunctional lycopene cyclase / phytoene synthase (KJ468785) at 38.4\% identity. The best BLASTX hit of the C. lativittatus transcript for this ORF against the SwissProt database was a “bifunctional lycopene cyclase/phytoene synthase” from Phycomyces blakesleeanus (UniProt accession Q9P854; e-val 3e-95). The m.15019 transcript has an FPKM value of 3.42. \\
\indent The third key enzyme in the β-carotene biosynthesis and metabolism is β-carotene 15,15’-monooxygenase (BCMO1). A FASTA search using the B. emersonii putative carotenoid dioxygenase sequence (KJ468787) identified two transcripts contained ORFs which were identified as putative homologs, m.16827 (670-aa, 44.2\% identity) and m.4639 (156-aa, 26.6\% identity). The top BLASTP hit against SwissProt for m.16827 was BCMO1 from Homo sapiens (UniProt accession Q9HAY6; e-val: 1e-44), and that for m.4639 was BCMO1 from Mus musculus (UniProt accession Q9JJS6; e-val: 3e-09). These transcripts had FPKM values of 2.57 and 0.997, respectively. \\
\indent To provide additional support for the candidate transcripts identified above, HMM profiles were generated from sequences available from the Kyoto Encyclopedia of Genes and Genomes (KEGG) database (Kanehisa and Goto 2000; Kanehisa et al. 2014). When available, the Metazoan, Eukaryote, and/or Plant genes were used. Otherwise, the bacterial and archaeal protein sequences were used (supTab\_KEGGHMM\_list). The candidate C. lativittatus transcripts above were also recovered from these HMM searches. \\
\indent Sequence searches identified all three of these β-carotene metabolism genes in the genomes of two Blastocladiomycota fungi, Allomyces macrogynus and Catenaria anguillulae. Similar searches of the genomes of the Chytridiomycota fungi Batrachochytrium dendrobatidis, Homolaphlyctis polyrhiza, and Spizellomyces punctatus, found an incomplete complement of these genes of this pathway. The B. dendrobatidis genome contains no homologs for any of these genes, while the H. polyrhiza genome contains a candidate phytoene desaturase homolog (top BLASTP hit against SwissProt: phytoene desaturase from P. blakesleeanus [P54982.1], e-val: 2e-68, 49\% identity), and S. punctatus possesses a candidate β-carotene oxygenase homolog (top BLASTP hit against SwissProt: β,β-carotene 9’,10’-oxygenase from Macaca fascicularis [Q8HXG8.2], e-val: 1e-37, 26\% identity). A comparative summary of these results is provided in (Figure\_bcaroPresenceAbsence). \\
\indent To assess the phylogenetic history of BCMO1, a maximum likelihood tree was constructed from homologs found in Ascomycete, Chytridiomycete, Blastocladiomycete, and Zygomycete species (FigS1\_BCMO1DO2). Examination of the resulting gene tree topology provides strong support for the early-diverging fungal genes to cluster distinctly outside the metazoan gene lineages, and suggests at least 3 major duplication events. At least one duplication occurred exclusively in the metazoan lineages to give rise to BCMO1 and BCDO2. One duplication likely occurred prior to the fungal/metazoan divergence, resulting in the copies seen in the Chlorophyta, Dikarya, Zygomycota. A second duplication likely occurred after the divergence of the fungi and prior to the divergence of the Cryptomycota, resulting in two subtypes of fungal β-carotene oxygenase. Interestingly, while copies of each subtype can be found in members of the Zygomycota, only one or the other is found in the Chytridiomycota and Blastocladiomycota.\\
\indent The resulting gene tree was reconciled with a non-binary species tree generated with NCBI using Notung (v2.6) (BCMO1DO2\_reconciledTree.png <messy figure, might not be ultimately necessary>). This analysis supports the events described above and suggests that there were multiple additional duplications in specific zygomycete lineages. \\
\subsection{Sensing: Photosensing capacity is consistent with ideas about C.lat abilities}
To determine the possible nature of observed photosensory capacity in C. lativittatus, we searched our transcriptome for ORFs predicted to be associated with photobiology (SupTab\_photosensing) using known fungal photobiology proteins, including opsins and opsin-like proteins, circadian rhythm proteins (WC-1 and 2, FRQ, and FWD-1), cryptochromes, phytochromes, and the photoreceptor protein VIVID. \\
\indent A search for homologs to Neurospora crassa White Collar-1 (WC-1, NCU02356) and White Collar-2 (WC-2, NCU00902) proteins identified a 368 aa ORF (m.12730) as a potential WC-1 homolog. They are reciprocal best-blast hits as NCU02356 is the top BLASTP hit against SwissProt using this m.12730 as a query (UniProt id: Q01371, e-val: 4e-43, 35.2\% identity). Additionally, m.12730 is predicted by InterPro to contain two PAS domains (IPR000014), similar to NCU02256. However, it is much shorter: only 368 aa compared to 1167 aa for NCU02256. Other WC-1 homologs were recovered from H. polyrhiza, S. punctatus, C. anguillalae, and A. macrogynus (PAS\_alignment). No ORFs were predicted as WC-2 homologs, nor were there any other potential PAS-domain containing transcripts. \\
\indent In addition to the white collar complex proteins WC-1 and WC-2, the blue-light sensitive photoreceptor protein VIVID (VVD) identified in N. crassa and other filamentous fungi is a small (186 aa), cytoplasmic flavoprotein that responds to increasing light intensity (Schwerdtfeger and Linden 2003). No putative homologs of the N. crassa VVD protein (NCU03967) were recovered in a BLASTP search against the C. lativittatus transcriptome. Additionally, no homologs were observed in the other Blastocladiomycete and Chytridiomycete species surveyed. This absence is consistent with an observed absence of VVD homologs outside of the Sordariomycete lineages. \\
\indent Phytochromes are, among other things, circadian rhythm regulators in plants (Rockwell et al. 2006), and Velvet A homologs are demonstrated to be regulators for secondary metabolism and sporulation in several fungi (Calvo 2008). There were no putative phytochrome homologs identified in C. lativittatus using an HMM generated from seed sequences in PFAM family PF00360, nor any homologs of the phytochrome-associated Velvet protein using either the N. crassa Velvet A-like protein NCU01731 or an HMM generated from N. crassa and additional Aspergillus and Fusarium sequences. While phytochromes are known to be present in the Chytridiomycete Spizellomyces punctatus (see (Idnurm et al. 2010), there were no homologs for Velvet or phytochrome observed in other members of the Chytridiomycota and Blastocladiomycota. \\
\indent A total of 6 ORFs were predicted to be opsin-related proteins based on predicted Pfam domains and predicted seven transmembrane helical domain architecture (SupTab\_photosensing). Of particular note is a predicted 537 aa ORF (m.7819), which has two identifiable Pfam domains: a 213 aa region with similarity to bacterial rhodopsin (PF01036; e-val: 4.6e-22), and a 178 aa region with similarity to guanylate cyclase (PF00211; e-val: 3.6e-51). This architecture is similar to that found in Allomyces macrogynus and Catenaria anguillalae, and described more fully in Blastocladiella emersonii (Avelar et al. 2014). A homolog was also identified in Homolaphlyctis polyrhiza. When compared with the other examples of this protein architecture found in the Blastocladiomycota and Chytridiomycota, this C. lativittatus transcript shares 61.91\%, 72.98\%, 71.19\%, 64.20\%, 63.36\%, and 54.55\% identity with the B. emersonii, each of the four A. macrogynus, and the H. polyrhiza proteins, respectively.  \\
\indent To ascertain the placement of this C. lativittatus Opsin-GC fusion protein among other opsin proteins, a maximum likelihood tree was generated using opsin sequences from Avalar et al., with additional inclusion of the opsin-GC fusion sequences recovered from H. polyrhiza, C. anguillalae, and C. lativittatus (OpsinGCFusion\_tree). The C. lativittatus and C. anguillalae sequences cluster expectedly with the other Blastocladiomycete sequences (A. macrogynus and B. emersonii) in a well supported early-diverging fungal group. Perhaps unexpectedly, the sequence from the Chytridiomycete H. polyrhiza falls within, rather than outside of, the Blastocladiomycete sequences, albeit with a relatively long branch (OpsinGCFusion\_chytridCluster).\\
\subsection{"interesting" animal homologs}
Initial BLAST results against the nr database revealed a number of hits with putative animal homologs. An attempt to classify these hits further was made using a search against the Swissprot database. Of the 10 genes initially identified, only 8 had SwissProt hits with e-val < e-10. The number of hits to these 8 transcripts varied considerably, ranging from 2 up to 2407 (Rob\_mammalian\_hits). Nonetheless, all hits scoring better than e-10 were collected, aligned to the C. lat transcript with Mafft, trimmed with trimal, and trees constructed with FastTree. (Trees are pretty messy) \\
%% Discussion
%%%%%%%%%%%%%%%
\section{Discussion}
Coelomomyces lativittatus is the only known insect pathogen among the early-diverging fungal lineages and has been well-studied as a potential mosquito control agent. This transcriptome study represents the first attempt at developing available genomic and proteomic resources for this and other Coelomomyces species. Future work will most assuredly expand on the results demonstrated here, including whole genome sequencing, developmental and life stage-specific RNA sequencing, and proteomic extraction and characterization. Nonetheless, some observations from the analyses performed here are useful in the comparative genomics of non-insect associated early-diverging fungi, and can also provide a focus for future work in C. lativittatus. \\
\indent Insect Virulence discussion: Check out previously described repertoire in other fungi to see what’s there. Surprising result is apparent unique expansion of C1 cysteine protease proteins unique to C. lativittatus, with previously inferred cuticle-degrading activity. \\
\indent While there are several examples of entomopathogenic fungi, C. lativittatus and other members of Coelomomyces are the only known members of the Blastocladiomycota which demonstrate this association, and the biological mechanism by which C. psorophorae infects mosquito larvae has been documented previously (Travland 1979a; Zebold et al. 1979). In general, infection is initiated by the settling of the spore onto the host cuticle, followed by encystment and secretion of thin cell wall. The appearance of an appressorium and subsequent development of a penetration tube which pierces the integument of the host then allows the fungus to enter the host hemocoel (Zebold et al. 1979). \\
\indent As noted by Travland (Travland 1979a) in C. psorophorae, there is a correlation between disruption of the outermost layer of the cuticle and accumulation of an amorphous, electron-dense material at the cuticle-contacting tips of penetration tubes. As the appressorium tip is the site of actual penetration through the cuticle and into the mosquito larvae, a speculative explanation of the observable electron-dense material would be the proteases and other degradation-related proteins unique to C. lativittatus recovered in this study. Indeed, a hypothesis postulated at the time (Travland 1979a) suggested that this material may be enzymatic in nature. \\
\indent A comparison of counts of the top 20 PFAM domains (Table1\_Clat\_top20\_comparison) suggests four protein families which appear to be uniqely expanded in C. lativittatus relative to the other non-insect associated Blastocladiomycete and Chytridiomycete species. These include “myosin tail” (PF01576), “glyco\_hydro\_47” (PF01532), “trypsin” (PF00089), and C1 peptidase (PF00112). Of these four, the latter two have clear protease and degradation functions. While the PF00112 phylogenetic history is a little unclear, at least two C. lativittatus specific groups can be identified, one of which appears to cluster with known arthropod sequences. An additional C. lativittatus sequence clusters with the plant sequences from papaya and pineapple, known to have culticle degrading activity in nematodes. Further work, especially life-stage dependent RNAseq experiments are completely necessary to confirm this hypothesis and would be critical in order to map expression levels of these and other protease genes before, during, and after infection. Aspects of infection which cannot be described further in this study are the adhesion vesicles, which are hypothesized to secrete the “glue” that attaches the spore to the host, and the observed pseudopodia structures, which appear after the settling of the spore. \\
\indent Searches for a 20-hydroxyecdysone receptor based on similarity to known arthropod receptors identified at least one candidate transcript with similarity to the DNA binding domain of the ecdysone receptor in D. melanogaster. This DNA binding domain profile, PFAM id: PF00105, is specifically associated with nuclear receptors. Sequence, structure, and phylogenetic analysis suggests that while it is significantly diverged in sequence, it may be a hormone receptor and is unlikely to be the result of a horizontal transfer event or sequence contamination. Furthermore, homologous sequences are not found in other Blastocladiomycete and Chytridiomycete species surveyed, providing a tantalyzing explanation of Coelomomyces observed affinity for 20-hydroxyecdysone from mosquitoes. However, we are hesitant to hold this as undeniable evidence of presence of this receptor in C. lativittatus; rather it is submitted as a starting point for future analyses. Further work to evaluate gene expression changes in this and other transcripts when C. lativittatus is exposed to the anophelid larvae or the 20-hydroxyecdysone hormone are absolutely crucial and will provide better insight into these candidate genes. \\
\subsection{Sensory results are consistent with previous hypotheses, but undetermined if this represents a specific insect association aspect.}
The presence of a moderately complex sensory network governing the full C. lativittatus life cycle can be inferred from experimental research on other Coelomomyces species. For example, C. psorophorae zygotes need to seek out Cu. inornata larvae (Whisler et al. 1975). Once infected, the zygotes must develop into sporangia, regulate the timing of meiospore release, and these meiospores need to find the crustacean host: the copepod Cy. vernalis in the case of C. psorophorae (Whisler et al. 1974). Once inside the crustacean host, similar regulation of sporangial development and spore release must also take place, but in a much different environment. Reared under identical conditions, dehiscence of C. dodgei and C. punctatus occurs at significantly different times (Federici 1983) suggesting the presence of a photoperiod dependent regulatory mechanism. These spores must then seek out members of the opposite mating type (Federici 1983), fuse to form zygotes, and exhibit phototactic capabilities to swim upwards to the water surface (Federici 1983). \\
\indent Given this evidence in other Coelomomyces species, C. lativittatus likely also possesses a complex sensory network relative to chytrids which display no insect association. The demonstrated ability for photoperiod regulation prompted our search for transcripts predicted to be involved in photosensing. From this search, one putative homolog of the N. crassa White collar-1 protein was identified. The remaining components of the white collar / circadian rhythm process (White collar-2, FRQ, FWD-1), however, were not recovered. The White collar-2 protein is present, however, in the chytridiomycete S. punctatus and the Blastocladiomycete C. anguillulae, but is absent in the other Chytridiomycete and Blastocladiomycete species surveyed. Therefore it is not necessarily unusual to find its incomplete presence in C. lativittatus, especially given the limitations of the current transcriptome study. \\
\indent Several proteins predicted to be opsins or opsin-related were identified based on transmembrane domain architecture and PFAM domain identification. Notably, one transcript is predicted to have a type 1 microbial rhodopsin domain fused with a guanylate cyclase domain. This structure is similar to a novel fusion protein recently described in the Blastocladiomycete Blastocladiella emersonii, and additional homologs can be identified in the genome assemblies of other Blastocladiomycetes A. macrogynus and C.  anguillulae (Avelar et al. 2014). The mechanism of activity of the fusion protein described for B. emersonii is that light activates the type 1 rhodopsin domain, which in turn activates the coupled GC domain. This facilitates synthesis of cGMP, which activates K+-selective cyclic nucleotide gated channels. Voltage-activated Ca2+ channels, activated by the resulting hyperpolarization of the plasma membrane, would elevate Ca2+ levels, prompting interaction with the flagellum and ultimately mediating phototaxis (Avelar et al. 2014). \\
\indent The presence of this fusion protein in C. lativittatus, in addition to its previously described presence in B. emersonii, A. macrogynus, and C. anguillulae, supports the hypothesis that the novel fusion gene appeared prior to the divergence of the Blastocladiomycota lineage as it can now be said to be present in all three of the Blastocladiaceae, Catenariaceae, and Coelomomycetaceae families. Furthermore, its presence in the Chytridiomycete H. polyrhiza suggests that the fusion appeared earlier. However the fusion architecture does not appear in any of the other Chytridiomycete genomes surveyed, suggesting that its presence in H. polyrhiza is the result of either a recent fusion event, duplication and losses in the other Chytridiomycete lineages, or HGT event.\\
\subsection{β-carotene biosynthesis and metabolism pathways are present and nearly complete in the C. lativittatus transcriptome.}
Endogenous β-carotene production is biologically important for many reasons, one of which is that it functions as precursor to retinal, the critical component of rhodopsin-mediated photoreception. Carotenoid production in the Blastocladiomycota is well known, and while γ-carotene is the predominant molecule in B. emersonii and several Allomyces species, β-carotene specifically is known to be produced by C. dodgei. In this and other Coelomomyces species, the relative levels of β-carotene are indicative of mating type, implying that either the production and/or regulation of β-carotene is at some level influenced by or related to the same mechanics which govern sexual reproduction. The extent of this relationship remains to be explored. \\
\indent The presence of nearly all critical enzymes in the retinal biosynthesis pathway in C. lativittatus is consistent with these previous observations about, and suggests a fairly straightforward biological mechanism for, β-carotene production in Coelomomyces. Additionally, the presence of this pathway, coupled with the identification of multiple opsin-related transcripts, suggests that Coelomomyces has the biochemical capacity for rhodopsin-mediated photoreception. \\
\indent However, the lack of a phytoene dehydrogenase homolog, the first step in β-carotene biosynthesis, is unusual given its presence in related Blastocladiomycetes A. macrogynus, B. emersonii, and C. anguillulae. This absence suggests that either this gene is not transcriptionally active during the life stage sampled, mRNA transcripts from this gene were not recovered at detectable levels during RNA extraction, or C. lativittatus uses a novel mechanism for conversion of phytoene to lycopene to produce β-carotene. \\
\indent The enzyme responsible for conversion of β-carotene to retinal is β-carotene monoxygenase. This is one of two enzymes capable of cleaving β-carotene, the other being β-carotene dioxygenase. Two transcripts were recovered bearing similarities to the predicted BCMO1 protein from  B. emersonii, and strong similarity to BCMO1 profiles generated from known metazoan sequences. A phylogenetic reconstruction positions these transcripts expectedly within a Blastocladiomycete specific group of mono-oxygenase homologs, itself within a fungal-specific group. This suggests at least one duplication event occurred after the divergence of Fungi from the metazoan lineages. \\
\indent These findings presented in this study are descriptive and represent the first insights into the deeper molecular biology of this insect pathogen. The extent of the proteins, pathways, and networks studied in this work can be elucidated more completely once an annotated genome and life-stage specific transcriptomes are generated. For example, the diversity and copy number of sensory proteins actually present in the genome is likely to be higher than those captured by this transcriptome study, and a clearer picture of β-carotene biosynthesis will almost certainly be observed. \\
\indent Nonetheless, this initial work still provides a perspective on the underlying biology in the single chytrid pathogen of insects. Near-term future work will deal with obtaining a draft reference genome from C. lativittatus, enhanced transcriptome sequencing accounting for important developmental timepoints, and proteomics studies dealing with surface receptors necessary for environment sensing. Potential long term applications of this and other work include exploitation as a means of mosquito population control. \\
