%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% Document: Thesis for PhD at UC Riverside                                    %%
%% Title: Investigating the evolution of environmental and biotic interactions %%
%%          in basal fungal lineages through comparative genomics              %%
%% Author: Steven Ahrendt                                                      %%
%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%%
%% FLAGELLAR ANALYSIS %
%%%%%%%%%%%%%%%%%%%%%%%
\chapter{Comparative genomics analysis of Flagellar motility}
\label{app:Flagella}
\section{Introduction}
The eukaryotic flagellum is a protenaceous structure which provides motility to single-celled organisms [citation needed]. In fungi, only members of the basal lineages (specifically the Cryptomycota, Chytridiomycota, Blastocladiomycota, and Neocallimastigomycota) posess a flagellum and motile zoosporic life stage. Members of the Glomeromycota, Basidiomycota, and Ascomycota do not have such a life stage [citation needed] and are commonly considered "terrestrial fungi". Notable exceptions to this pattern include the Microsporidian lineage, which is considered a member of the basal fungi but whose members are not flagellated, and \textit{Olpidium brassicae}, a flagellated, single-celled organism morphologically similar to other chytrids but phylogenetically similar to terrestrial fungi \cite{Sekimoto2011}. \\
\section{Results and Discussion}
A search for Rozella homologs of flagellar-associated proteins from the \textit{Naeglaria} genome \cite{FritzLaylin2011} reveal a pattern of presence/absence of proteins in Rozella which correlates with that found in the Chytridiomycota and Blastocladiomycota. This pattern, in general, differs from the Microsporidia, supporting the placement of Rozella as prior to the Chytridiomycota/Blastocladiomycota and a flagellar loss event after the divergence of the Microsporidia. \\
\indent The heatmap displayed in Figure~\ref{fig:AppFlag_heatmap} indicates presence or absence of proteins in the \textit{Naeglaria} dataset as identified in the proteomes of other fungi searched. By clustering based on presence/absence patterns, the flagellated vs non-flagellated organisms separate into distinct groups consistent with known phylogeny (ie basal lineages separate from Dikarya and Zygomycetes). It's possible to collect a subset of genes which are present in basal (flagellated) lineages and absent in terrestrial (non-flagellated) lineages. Presumably, these have more pronounced association with flagellar assembly and function. Furthermore, take the subset of proteins present in all lineages and do pairwise distance alignment to see if, while being from the same family / hits to same query, these genes mutated or transititioned to new structures / functionalities over evolutionary time.\\
\section{Methods}
The flagellar analysis was carried out using a dataset of 173 flagellar motility proteins obtained from the \textit{Naeglaria grubeii} genome sequence \cite{FritzLaylin2011}. These proteins were used in a FASTA search (SSEARCH v36.07) using an e-val cutoff of e-20. Heatmap generation was accomplished using $pheatmap()$ in R.\\
\indent Multiple sequence alignment and distance matrix generation was performed on the subset of genes present in all organisms using $tcoffee$ and $phylip$.\\
